src/sentry/tempest/tasks.py

```diff
@@ -26,9 +26,15 @@ def poll_tempest(**kwargs):
     # FIXME: Once we have more traffic this needs to be done smarter.
     for credentials in TempestCredentials.objects.all():
         if credentials.latest_fetched_item_id is None:
-            fetch_latest_item_id.delay(credentials.id)
+            fetch_latest_item_id.apply_async(
+                kwargs={"credentials_id": credentials.id},
+                headers={"sentry-propagate-traces": False},
+            )
         else:
-            poll_tempest_crashes.delay(credentials.id)
+            poll_tempest_crashes.apply_async(
+                kwargs={"credentials_id": credentials.id},
+                headers={"sentry-propagate-traces": False},
+            )
```

Comment:
Not part of this PR, but I noticed the task function does not accept `**kwargs`. See https://develop.sentry.dev/backend/application-domains/asynchronous-workers/: 

> Tasks must accept `**kwargs`` to handle rolling compatibility.

There's some more guidelines in there as well.

Reply:
It's not a problem in this case because we only pass `kwargs` that the task happens to accept, is that it?

Reply:
That is a pretty cool doc that I was not aware of.